src/testing/platform.ts

```diff
@@ -1,9 +1,10 @@
-import { getPlatform, setPlatform } from '@builder.io/qwik';
+import { getPlatform, QRL, setPlatform } from '@builder.io/qwik';
 import type { TestPlatform } from './types';
 import { existsSync } from 'fs';
 import { fileURLToPath } from 'url';
+import { qExport } from '../core/import/qImport';
 
-export function setTestPlatform(document: any) {
+function createPlatform(document: Document) {
```

Comment:
I left it as an `any` here cuz I'm not sure if like jsdom or domino's types would error against the lib.dom `Document`.